src/mmw/js/src/compare/templates/compareInputs.html

```diff
@@ -3,3 +3,8 @@
     <button id="compare-input-button-chart" class="{{ 'active' if mode == 'chart' }}"><i class="fa fa-bar-chart"></i></button>
     <button id="compare-input-button-table" class="{{ 'active' if mode == 'table' }}"><i class="fa fa-table"></i></button>
 </div>
+<div class="compare-download compare-input">
+    <button id="compare-input-button-download">
```

Comment:
I think the button should be disabled while polling — this will prevent the case where there's a new precipitation value in the csv, shown with stale results.

Reply:
Good idea -- cc6eb27 handles that and adds a missing unit value to the headers.